app/src/examples/AnimatedSensorMagneticFieldExample.tsx

```diff
@@ -0,0 +1,39 @@
+import Animated, {
+  SensorType,
+  useAnimatedSensor,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+import { StyleSheet, View } from 'react-native';
+
+import React from 'react';
+
+export default function AnimatedSensorMagneticFieldExample() {
+  const gravity = useAnimatedSensor(SensorType.MAGNETIC_FIELD);
+
+  const animatedStyle = useAnimatedStyle(() => {
+    const { x, y } = gravity.sensor.value;
+    const angle = (Math.atan2(y, x) * 180) / Math.PI;
```

Comment:
```suggestion
    const angle = Math.atan2(y, x) * 180 / Math.PI;
```

Reply:
According to prettier, this statement should be wrapped with brackets.